plugins/catalog-backend-module-bitbucket-server/src/providers/BitbucketServerEntityProvider.ts

```diff
@@ -106,15 +105,21 @@ export class BitbucketServerEntityProvider implements EntityProvider {
         options.schedule ??
         options.scheduler!.createScheduledTaskRunner(providerConfig.schedule!);
 
+      const defaultBranch = providerConfig.defaultBranch;
+      if (defaultBranch === undefined)
```

Comment:
suggestion: use "master" as default as in other providers and make this config option optional

Reply:
In my next push I am going to remove it as a config variable overall and store it in the location entity. So I will mark this as resolved regardless.